modules/olmv1-version-range-support.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="olmv1-version-range-support_{context}"]
+= Support for version ranges
+
+In {olmv1-first} you can specify a version range by using a comparison string in an Operator or extension's custom resource (CR). If you specify a version range in an Operator's CR, {olmv1} installs or updates the latest version of the Operator that can be resolved within the version range.
```

Comment:
```suggestion
In {olmv1-first} you can specify a version range by using a comparison string in an Operator or extension's custom resource (CR). If you specify a version range in an Operator's CR, {olmv1} installs or updates to the latest version of the Operator that can be resolved within the version range.
```
? Nit, but, since "the latest version" itself isn't being updated.